npm/cypress-schematic/sandbox13/README.md

```diff
@@ -0,0 +1,27 @@
+# Sandbox
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 12.0.0.
```

Comment:
The title of the file (sandbox13) suggests this is angular 13, but the readme says angular 12?